src/executables/util_cs.c

```diff
@@ -4548,6 +4548,7 @@ memmon (UTIL_FUNCTION_ARG * arg)
 {
 #if defined(CS_MODE)
   constexpr int DEFAULT_OPTION_PRINT_CNT = 5;
+  constexpr int MODULE_INDEX_MAGIC_NUMBER = -2097573308;	/* INT_MIN + 49910340 */
```

Comment:
We need a comment if something is not self-explanatory. What's it for?

Reply:
I actually think that setting a magic number to a meaningful value is more risky. 49910340 is my birthday reversed within a certain range :)